todo.css

```diff
@@ -0,0 +1,36 @@
+body {
+  background-color: #FAECA9
+}
+
+.topBar {
+  font-family: sans-serif;
+  font-size: 1em;
+  text-align: center;
+  color: white;
+  position: fixed;
+  top:-3%;
```

Comment:
you mean lines 10 and 11? So you mean change it so that the top bar doesn't stay on the screen when scrolling? Any particular reason for this?